boards/sim/sim/sim/configs/segger/defconfig

```diff
@@ -16,6 +16,7 @@ CONFIG_BOARDCTL_POWEROFF=y
 CONFIG_BOARD_LOOPSPERMSEC=0
 CONFIG_BOOT_RUNFROMEXTSRAM=y
 CONFIG_BUILTIN=y
+CONFIG_CONSOLE_RTT=y
```

Comment:
all change except this one need move to the first patch

Reply:
done